src/frontend/admin/src/components/templates/products/form/sections/target-courses/ProductFormTargetCourseRow.tsx

```diff
@@ -37,6 +47,13 @@ export function ProductFormTargetCourseRow({ item, ...props }: Props) {
           {item.course.title}
         </CustomLink>
       }
+      permanentRightActions={
+        item.is_graded ? (
+          <Tooltip title={intl.formatMessage(messages.isGradedTooltip)}>
+            <GppGoodIcon />
+          </Tooltip>
+        ) : undefined
+      }
```

Comment:
Is there a props to put it on left side ? I would prefer to keep the right side for action not information.

Reply:
I also asked myself the question, but I chose on the right because at least we have the same position all the time, while on the right, it will depend on the size of the titles